app/helpers/application_helper.rb

```diff
@@ -1729,7 +1729,7 @@ def route_exists?(hash)
 
   def translate_header_text(text)
     if text == "Region"
-      I18n.t("product.name") + " " + _(text)
+      Vmdb::Appliance.PRODUCT_NAME + " " + _(text)
```

Comment:
Is there any particular reason why only this one occurrence of `I18n.t('product.name')` was changed?
I'm asking since we use that all over the place.

Reply:
I did a search and could only find that one! I'll have a look again then to see where the others are. Thanks @mzazrivec

Reply:
Ahha, I was searching for `I18n.t("product.name")` but seems many uses are using single quotes `I18n.t('product.name')` :)